config/configstore/model.go

```diff
@@ -546,12 +546,11 @@ func (acc *Account) GetEthereumContextWaitTimeout() time.Duration {
 
 // SignMsg signs a message with the signing key
 func (acc *Account) SignMsg(msg []byte) (*coredocumentpb.Signature, error) {
-	//TODO change signing keys to curve ed25519
 	keys, err := acc.GetKeys()
 	if err != nil {
 		return nil, err
 	}
-	signature, err := crypto.SignMessage(keys[identity.KeyPurposeSigning].PrivateKey, msg, crypto.CurveEd25519, true)
+	signature, err := crypto.SignMessage(keys[identity.KeyPurposeSigning].PrivateKey, msg, crypto.CurveSecp256K1, true)
```

Comment:
I think we should get rid of `crypto.SignMessage` long term and directly call the secp package or at least have a distinct way to do ethereum signatures that hash the message first.